README.md

```diff
@@ -12,6 +12,7 @@ versioning: ![versioning](https://img.shields.io/badge/calver-YYYY.MM.PATCH-22bf
 * [Browser Stack](docs/05-browser-stack.md)
 * [Delivery](docs/03-delivery.md)
 * [CI/CD](docs/06-cicd.md)
+* [WSL Setup](docs/07-wsl-setup.md)
```

Comment:
Maybe rather link this in `Getting Started`. Not sure if a separate point is necessary and good here since it is not relevant for everybody.